lib/konfa.rb

```diff
@@ -112,6 +112,7 @@ def init
       end
 
       def init_with(suffix, *args)
+        self.reinit
```

Comment:
We we should do this only if it isn't initialized. I know it won't have any effect with the current implementation, but that is the expected behaviour. That would avoid mishaps for any subclasses hooking in to the reinit-call